src/v3/src/util/formUtils.ts

```diff
@@ -249,17 +242,11 @@ export const getIdpButtonElements = (
     return {
       type: 'Button',
       label: displayName,
+      noTranslate: type === 'general-idp',
       options: {
         type: ButtonType.BUTTON,
         step: IDX_STEP.PIV_IDP,
         dataSe: 'piv-card-button',
-        classes: classNames(
-          'social-auth-button',
-          `social-auth-${type}-button`,
-          // @ts-expect-error OKTA-609464 - className missing from IdpConfig type
-          idpObject.idp?.className,
-          { 'no-translate': type === 'general-idp' },
-        ),
```

Comment:
Would appending `idpObject.idp?.className` to `dataSe` also apply here @glenfannin-okta?

Reply:
And also maybe adding `social-auth-${type}-button` too?

Reply:
Yea, that would be good, or you could use `social-auth-${type}-button` if `idpObject.idp?.className` doesn't exist/isn't set